pos_user_restriction/security/pos_security.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="group_assigned_points_of_sale_user" model="res.groups">
+        <field name="name">User: Assigned POS Only</field>
+        <field name="category_id" ref="base.module_category_point_of_sale"/>
+        <field name="comment">The user will have access to her assigned points of sale.</field>
+    </record>
+    <record id="point_of_sale.group_pos_user" model="res.groups">
+        <field name="implied_ids" eval="[(4, ref('group_assigned_points_of_sale_user'))]"/>
+    </record>
+
+<data noupdate="1">
+    <record id="pos_config_assigned_users" model="ir.rule">
+        <field name="name">Assigned points of sale</field>
+        <field ref="point_of_sale.model_pos_config" name="model_id"/>
+        <field name="domain_force">['|',('assigned_user_ids','in',user.id),('assigned_user_ids','=',False)]</field>
+        <field name="groups" eval="[(4, ref('pos_restrict_users.group_assigned_points_of_sale_user'))]"/>
```

Comment:
```suggestion
        <field name="groups" eval="[(4, ref('group_assigned_points_of_sale_user'))]"/>
```

Since the module rename, make sure you update these kind of references :)

Reply:
The best is to always keep them relative (as the suggestion makes), for avoiding to be affected by renamings.

Reply:
oops, sorry, fixed